src/Algebra/Graph/Labelled/AdjacencyMap.hs

```diff
@@ -222,12 +218,11 @@ infixl 5 >-
 -- 'edgeCount'   (overlay 1 2) == 0
 -- @
 --
--- Note: 'overlay' composes edges in parallel using the operator '<+>' with
--- 'zero' acting as the identity:
+-- Note: 'overlay' composes edges in parallel using the operator '<>':
 --
 -- @
--- 'edgeLabel' x y $ overlay ('edge' e x y) ('edge' 'zero' x y) == e
--- 'edgeLabel' x y $ overlay ('edge' e x y) ('edge' f    x y) == e '<+>' f
+-- 'edgeLabel' x y $ overlay ('edge' e x y) empty == e
```

Comment:
Overlaying with `empty` doesn't seem to make much sense here. I think I would actually prefer to keep this comment talking about `<+>` and `zero`, for symmetry with the one below, saying something like "Assuming that 'zero' exists...".

Reply:
Also, if you're changing examples here, you should update them in the testsuite too.